openapi.yaml

```diff
@@ -478,6 +525,15 @@ components:
         paging:
           $ref: '#/components/schemas/Paging'
 
+    ExternalOrderRequest:
+      type: object
+      required:
+        - jwt
+      properties:
+        jwt:
+          type: string
+          description: string serialized JWTBodySpend
+
     OpenOrder:
```

Comment:
need to add to openOrder amount and recipient wallet address

Reply:
👍